src/models/closeButton.js

```diff
@@ -35,28 +37,22 @@ const closeButtonModel = {coeffs: [1,1,1,1,1,1,1,1],
                 return trapezoid(ratio, lowerBound, upperBound) ** coeffSquare;
             }
 
-            // Function isn't working; How can I tell the diference between a 'div' and 'button' tag?
-            function onClick(fnode) {
-                //console.log(fnode.element.tagName == 'button');
-                return ((fnode.element.hasAttribute("onclick")) ? ONEISH : ZEROISH) ** coeffOnClick;
-            }
-
             function topHalfPage(fnode) {
                 const rect = fnode.element.getBoundingClientRect();
                 const windowHeight = window.innerHeight;
 
                 return ((rect.top <= windowHeight/2) ? ONEISH : ZEROISH) ** coeffTopHalfPage;
             }
             
-            function largeZIndex(fnode) {
-                const lowerBound = 0;
-                const upperBound = 1000;
+            // function largeZIndex(fnode) {
+            //     const lowerBound = 0;
+            //     const upperBound = 1000;
 
-                return trapezoid(fnode.element.style.zIndex, lowerBound, upperBound) ** coeffZIndex;
-            }
+            //     return trapezoid(window.getComputedStyle(fnode.element).getPropertyValue("z-index"), lowerBound, upperBound) ** coeffZIndex;
+            // }
 
             function absolutePosition(fnode) {
-                return ((fnode.element.style.position == "absolute") ? ONEISH : ZEROISH) ** coeffAbsolutePosition;
+                return ((window.getComputedStyle(fnode.element).getPropertyValue("position") == "absolute") ? ONEISH : ZEROISH) ** coeffAbsolutePosition;
```

Comment:
Functionally fine, but watch out for == in the future; it has some weird corner cases. You're safer always using ===.